src/mail.sh

```diff
@@ -46,10 +46,39 @@ function mail_setup()
   return 0
 }
 
+function mail_verify()
+{
+  local count=0
+  local -a missing_conf
+  local -a min_conf=('user.name' 'user.email' 'sendemail.smtpencryption'
```

Comment:
min stands for minimal, right? Maybe we can call this `essential_config_options_list` or similar.